### PR TITLE
Slack integration

### PR DESCRIPTION
**Pull request by meeunier**
Write article for the slack integration. 

See https://api.slack.com/docs/slack-apps-checklist for requirements

### CODE REVIEW

**Comment by meeunier** [comment]
The content for the Slack integration support article is complete.

Feel free to tweak formatting (it's a little messy up there)

**Comment by meeunier** [comment]
@aetrion/dnsimple please take a 👀 and remove nonsense/add sense. Thx.

**Comment by jacegu** [comment]
Missing the explanation about the plans this feature is available to.

**Comment by meeunier** [comment]
Right. On it!

File: content/articles/slack-integration.markdown [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+![Slack Integration Step 3](/files/slack-integration-step-3.png)
+
+### Step 4: Select the DNSimple account you want to use.
+
+
+Start Slacking with the `/dnsimple` commands.
+
+## Domains commands
+
+### check if a domain is available
```
Comment (jacegu) [inline content/articles/slack-integration.markdown]: Shouldn't these subtitles start with a Capital letter?

**Comment by meeunier** [comment]
Passes the Visage test (it's a Litmus test with a twist 🍭 )

Feel free to keep on tweakin'

**Pull request merged**